guides/common/modules/proc_enabling-the-installer-managed-dns-service.adoc

```diff
@@ -0,0 +1,31 @@
+[id="enabling-the-installer-managed-dns-service"]
+= Enabling the installer-managed DNS service
+
+If you do not have a DNS server available in your network, you can use the installer-managed DNS service. This feature enables you to provide a DNS service with a low maintenance effort.
+
+Perform the steps on the {Project} or {SmartProxyServer} that you want to configure to manage the DNS service for the domain.
+
+
+.Procedure
+
+. Backup the answer files:
++
+[options="nowrap",subs="+quotes,attributes"]
+....
+# mkdir /root/backup-answer-files/
+# cp -p /etc/foreman-installer/scenarios.d/*-answers.yml /root/backup-answer-files/
+....
+
+. Configure {Project} or {SmartProxy} as DNS server:
++
+[options="nowrap",subs="+quotes,attributes"]
+....
+# {foreman-installer} \
+--foreman-proxy-dns-managed=true \
+--foreman-proxy-dns-provider=nsupdate \
+--foreman-proxy-dns-server="127.0.0.1" \
+--foreman-proxy-dns=true
```

Comment:
Nit: in other places we don't use `=`. Perhaps a good chance to make it consistent? I'd also list `--foreman-proxy-dns`, then `--foreman-proxy-dns-provider` and then any provider specific options. So:
```suggestion
--foreman-proxy-dns true \
--foreman-proxy-dns-provider nsupdate \
--foreman-proxy-dns-managed true \
--foreman-proxy-dns-server "127.0.0.1"
```

Then do the same for all commands. This should help make the pattern more visible.

Reply:
I removed the `=` in all options of foreman-installer commands.